analyses/cnv-chrom-plot/cn_status_heatmap.Rmd

```diff
@@ -271,31 +279,39 @@ Run the bin status calling on each sample.
 # Get a vector of the biospecimen IDs
 sample_ids <- unique(seg_data$ID)
 
-# Run call_bin_status for each biospecimen's segments.
-bin_calls_list <- lapply(sample_ids,
-  call_bin_status,
-  bin_ranges = bins,
-  seg_ranges = filtered_seg_ranges,
-  uncallable_ranges = uncallable_ranges,
-  frac_threshold_val = frac_threshold,
-  frac_uncallable_val = frac_uncallable
-)
+# Read in the calculation file if it was already made and this isn't the final figure
+if (file.exists(file.path(results_dir, "cn_status_bp_per_bin.tsv")) & !params$final_figure) {
```

Comment:
Can someone double check this logic for me? I basically am trying to make sure when we run this for the "final_figure" (see generate_figures.sh adds) that this always recalculates.

Reply:
Yep, on it.

Reply:
TY!

Reply:
This hurt my brain way more than it should have. It seems right to me though - 

![IMG_1922](https://user-images.githubusercontent.com/4701111/82684295-59574700-9c20-11ea-8d8b-4b793b0fd0f9.png)

Reply:
Yes! This makes sense to me! Thanks for helping confirm!